config/config.go

```diff
@@ -49,57 +51,20 @@ type Config struct {
 	API       *APIConfig
 }
 
-//ReadConfig loads the config from file
-func ReadConfig(confPath string, logger *zap.Logger) (*Config, error) {
-	// load the default config, if confPath not provided
-	if confPath == "" {
-		confPath = GetDefaultConfPath()
-	}
-
-	// Parse config file
-	conf := Config{
-		IKS: &IKSConfig{}, // IKS block may not be populated in secrete toml. Make sure its not nil
-	}
-	logger.Info("parsing conf file", zap.String("confpath", confPath))
-	err := ParseConfig(confPath, &conf, logger)
-	return &conf, err
-}
-
-// GetConfPath get configuration file path
-func GetConfPath() string {
-	if confPath := getEnv("SECRET_CONFIG_PATH"); confPath != "" {
-		return filepath.Join(confPath, "libconfig.toml")
-	}
-	//Get default conf path
-	return GetDefaultConfPath()
-}
-
-// GetConfPathDir get configuration  dir path
-func GetConfPathDir() string {
-	if confPath := getEnv("SECRET_CONFIG_PATH"); confPath != "" {
```

Comment:
Please make sure these methods are unused.

GetConfPathDir,  GetDefaultConfPath, GetConfPath etc.

Reply:
Only https://github.com/IBM/ibm-vpc-file-csi-driver/blob/d89edd7b27f251b84b17c002c1294bab0abd4f3f/cmd/main.go#L95 - here this is getting used, but will be updating file csi driver code also soon.